flow-typed/npm/react-redux_v5.x.x.js

```diff
@@ -0,0 +1,261 @@
+// flow-typed signature: 502cfd4f5e95c6308f747cdf16dc93ce
+// flow-typed version: 1751d5bf0a/react-redux_v5.x.x/flow_>=v0.68.0
+
+declare module "react-redux" {
```

Comment:
Do these need to be committed?

Reply:
Yes, it seems so:

<img width="868" alt="image" src="https://user-images.githubusercontent.com/16100194/48002183-12236700-e10b-11e8-9e47-afab4076c071.png">
(https://flow.org/blog/2016/10/13/Flow-Typed/)

Reply:
Or at least, it's recommended, but I suppose we can choose not to.

Reply:
Cool